library/src/main/java/com/novoda/downloadmanager/lib/DatabaseHelper.java

```diff
@@ -150,6 +151,19 @@ private void createDownloadsByBatchView(SQLiteDatabase db) {
         );
     }
 
+    private void createDownloadsByStatusView(SQLiteDatabase db) {
+        db.execSQL("DROP VIEW IF EXISTS " + DownloadContract.DownloadsByStatus.VIEW_NAME_DOWNLOADS_BY_STATUS);
```

Comment:
In order not to be able to observe a view that doesn't update as frequently as it does at the moment. (because the current bytes constantly change).